src/dialog/dialog.component.ts

```diff
@@ -116,6 +120,10 @@ export class Dialog implements OnInit, AfterViewInit, OnDestroy {
 
 		const parentElement = this.dialogConfig.parentRef.nativeElement;
 
+		this.animationFrameSubscription = this.animationFrameService.tick.subscribe(() => {
+			this.placeDialog();
+		});
+
```

Comment:
Also gotta wrap this in a conditional because the service is optional ...

```suggestion
		if (this.animationFrameService) {
			this.animationFrameSubscription = this.animationFrameService.tick.subscribe(() => {
				this.placeDialog();
			});
		}
```